### PR TITLE
new examples section which holds all examples, tutorials and videos

### PR DESCRIPTION
**Pull request by thisisjofrank**
new examples section which holds all examples, tutorials and videos

Sits under /examples/

image

image

image

image

image

### CODE REVIEW

File: examples/_pages/VideoPage.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { VideoReference } from "../types.ts";
+
+type Props = { video: VideoReference };
+
+export default function VideoPage({ video }: Props) {
+  return (
+    <>
+      <EmbedVideo id={video.externalURL} />
+      <div className="my-12">
+        Find more videos in the <a href="/examples/">Exampples page</a>{" "}
```
Comment (ry) [inline examples/_pages/VideoPage.tsx:11]: typo

**Comment by ry** [comment]
Video pages ([like this](https://deno-docs--new-examples.deno.dev/examples/what_is_deno/)) are lacking the "Prev" and "Next" links at the bottom

Screenshot 2024-12-10 at 1 38 32 PM

**Comment by thisisjofrank** [comment]
> Video pages ([like this](https://deno-docs--new-examples.deno.dev/examples/what_is_deno/)) are lacking the "Prev" and "Next" links at the bottom

I'd kind of like to remove the next and prev completely. I don't think they make sense, the content doesn't really flow in an order. What do you think @ry ?

File: examples/_pages/LandingPage.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      <div className="fully-filtered">
+        <h2 class="text-2xl font-semibold sm:text-3xl md:text-4xl">
+          Oops! You've filtered everything
+        </h2>
+        <p class="md:text-lg mt-4">
+          Maybe remove a filter to see some examples?
+        </p>
+        <img
+          src="/examples/tutorials/images/deno-educator.png"
+          alt="deno on a skateboard"
```
Comment (thisisjofrank) [inline examples/_pages/LandingPage.tsx:101]:
```suggestion
          alt="deno on a skateboard"
```
```suggestion
          alt="sad Deno with a grad cap"
```

**Comment by ry** [comment]
> I'd kind of like to remove the next and prev completely. I don't think they make sense, the content doesn't really flow in an order. What do you think @ry ?

ya that makes sense

File: examples/_pages/examples_test.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+//     await t.step("Check graph: " + path, async () => {
+//       const result = await new Deno.Command(Deno.execPath(), {
+//         args: ["info", path],
+//       }).output();
+//       assertEquals(result.code, 0);
+//       assertNotMatch(decoder.decode(result.stdout), /\(resolve error\)/);
+//     });
+//   }
+// });
```
Comment (josh-collinsworth) [inline examples/_pages/examples_test.ts:21]: Should this be here? Seems like we should either write tests or delete the file.

**Review by josh-collinsworth — APPROVED** [review]
I added a few small visual updates of my own, just because I was in the repo already and figured it would be easier than writing out comments. Happy to discuss or update if there's an issue. But other than that and one non-blocking comment, LGTM 👍

File: examples/_pages/examples_test.ts (hunk 3 above)
Comment (thisisjofrank) [reply examples/_pages/examples_test.ts:21]: Ah dammit. I completely forgot about the tests. Will double check tomorrow whether they work or not!

Comment (thisisjofrank) [reply examples/_pages/examples_test.ts]: fixed!

**Pull request merged**